pkg/controller/useraccountstatus/useraccount_status_controller_test.go

```diff
@@ -207,7 +208,7 @@ func newUserAccount(userName, resourceVersion string) *toolchainv1alpha1.UserAcc
 	return userAcc
 }
 
-func newMasterUserRecord(userName, syncIndex string) *toolchainv1alpha1.MasterUserRecord {
+func newMasterUserRecord(userName, syncIndex string) *toolchainv1alpha1.MasterUserRecord { //nolint:unparam
```

Comment:
may I know why do we ignore linter here?

Reply:
because the linter says that it's always the same value that is used when this func is called, but since it's in a test, I guess it's ok and we can ignore the problem. I wanted to focus on the "production code" errors in this PR.

Reply:
ah, sure. because it's been always called with the userName = "johny" so it says that the param is not necessary. 
I understand that you want to focus on the "production code" but I wouldn't consider the tests as something less important. At the end of the day, we could set the linters to ignore all test files, but this is something that we really don't want to do I believe. 
My point is that it looks weird that it has the ignore clause when there is no problem with the function. I would rather change one of the callers instead of ignoring it.

Reply:
ok, fair enough, I'll update the test func, then. Or use another value in some tests 😉